src/app/pages/manage-buyer-and-both-requests/view-pending-verification/view-pending-verification.component.ts

```diff
@@ -173,18 +173,30 @@ export class ViewPendingVerificationComponent implements OnInit {
                   f.event?.toUpperCase() == 'ORGROLEUNASSIGNED' ||
                   f.event?.toUpperCase() == 'ADMINROLEASSIGNED' ||
                   f.event?.toUpperCase() == 'ADMINROLEUNASSIGNED'
-                ) {
-                  this.translate
-                    .get(f.event)
-                    .subscribe((val) => (f.event = val));
-                  if (f.event.includes('[RoleName]')) {
-                    var role = f.role;
-                    f.event = f.event.replace('[RoleName]', role);
+                ) 
+                {
+                  this.translate.get(f.event).subscribe((val) => (f.event = val));
+                  if (f.event.includes('[RoleName]')) 
+                  {
+                      if(f.roleKey != 'JAEGGER_SUPPLIER' &&
+                        f.roleKey != 'ACCESS_JAGGAER' &&
+                        f.roleKey != 'CAT_USER' &&
+                        f.roleKey != 'ACCESS_CAAAC_CLIENT' &&
+                        f.roleKey != 'JAEGGER_BUYER' &&
+                        f.roleKey != 'JAGGAER_USER')
+                      {
+                        var role = f.role;
```

Comment:
No need to declare `var role = f.role;` in 2 times

Reply:
Changes done